### PR TITLE
Adding Sample of Find-and-Replace text with data from database

### PR DESCRIPTION
**Pull request by sharonaumao**
Added KB documentation sample. The sample is:

- Replace-text-with-data-from-database

### CODE REVIEW

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.Drawing;
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs]: @Ramaraj-Marimuthu -Sorry I didn't see it but I have removed it.

File: Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        {
+            using (FileStream inputStream = new FileStream(Path.GetFullPath(@"../../../Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Set datasource and database values.
+                string datasource = "";
+                string database = "";
+                //Load the file stream into a Word document.
+                using (WordDocument document = new WordDocument(inputStream, FormatType.Automatic))
+                {
+                    SqlConnection SqlConn = new SqlConnection("Data Source=" + datasource + ";Initial Catalog=" + database + ";Integrated Security=True");
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs]: @Ramaraj-Marimuthu -I have rectified it Sir

File: Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                    //Retrive data from the table 'FindReplace' using SqlCommand.
+                    SqlCommand sqlCommand = new SqlCommand("Select * from FindReplace");
+                    sqlCommand.Connection = SqlConn;
+
+                    //Load the data into DataTable using SqlDataAdapter.
+                    SqlDataAdapter da = new SqlDataAdapter(sqlCommand);
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs]: @Ramaraj-Marimuthu -It is changed

File: Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                    da.SelectCommand.CommandTimeout = 0;
+                    DataTable dt = new DataTable();
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs]: @Ramaraj-Marimuthu -I have modified it

File: Find-and-Replace/Replace-text-with-data-from-database/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# How to replace text with data from database
+This example shows how to find a text in a Word document and replace it with a text from the database.
+
+# How to run this Sample
+
+1. Download this project to a location in your disk.
+
+2. Create database with column "TextToFind" and "ReplacementText" as the screenshot shown below.
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-text-with-data-from-database/README.md:8]: Create database with column "TextToFind" and "ReplacementText" as like the screenshot shown below.

File: Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            using (FileStream inputStream = new FileStream(Path.GetFullPath(@"../../../Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Set datasource and database values.
+                string datasource = "";
+                string database = "";
+                //Load the file stream into a Word document.
+                using (WordDocument document = new WordDocument(inputStream, FormatType.Automatic))
+                {
+                    SqlConnection sqlConnection = new SqlConnection("Data Source=" + datasource + ";Initial Catalog=" + database + ";Integrated Security=True");
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs:21]: //Create sql connection to get data from database

File: Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+                    //Retrive data from the table 'FindReplace' using SqlCommand.
+                    SqlCommand sqlCommand = new SqlCommand("Select * from FindReplace");
+                    sqlCommand.Connection = sqlConnection;
+
+                    //Load the data into DataTable using SqlDataAdapter.
+                    SqlDataAdapter dataAdapter = new SqlDataAdapter(sqlCommand);
+                    dataAdapter.SelectCommand.CommandTimeout = 0;
+                    DataTable dataTable = new DataTable();
+                    dataAdapter.Fill(dataTable);
+                    //Find and replace text with other text from SQL server.
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs:31]: Modify the comment as
//Iterate DataRow to find and replace text with other text from database

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: Find-and-Replace/Replace-text-with-data-from-database/README.md (hunk 5 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-data-from-database/README.md:8]: @MohanaselvamJothi -I have modified it

File: Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs (hunk 6 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs:21]: @MohanaselvamJothi -I have added the comment

File: Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs (hunk 7 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs:31]: @MohanaselvamJothi -I have modified the comment

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**